salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -239,7 +277,7 @@ spec:
           service:
             name: dex
             port:
-              number: 5554
+              number: 5556
```

Comment:
I think you forgot to re-render (after applying the manual patch on the service)

Reply:
Fixed !